cli/cmd/pack/pack.go

```
@@ -157,3 +90,93 @@ func (a *Agent) Build(ctx context.Context, opts *docker.BuildOpts, buildConfig *
 
 	return sharedPackClient.Build(ctx, buildOpts)
 }
+
+func getBuildpackName(ctx context.Context, bp string) (string, error) {
+	if bp == "" {
+		return "", errors.New("please specify a buildpack name")
+	}
+
+	u, err := url.Parse(bp)
+	if err != nil {
+		return bp, nil
```

do we want to return "", err here?

---

Nah, the original code actually ignores errors if its not a url and uses the string as is.